core/common/src/GeoCoordinateServices.ts

```diff
@@ -113,3 +116,20 @@ export interface GeoCoordinatesResponseProps {
   geoCoords: PointWithStatus[];
   fromCache: number;    // the number that were read from the cache rather than calculated.
 }
+
+/** Information required to interpret or complete a Geographic CRS in the specified format.
+ *  The only currently supported formats are JSON (for completion) and WKT (OGC Well Known Text)
+ * @beta
+ */
+export interface GeographicCRSInterpretRequestProps {
+  format: "WKT" | "JSON";
+  geographicCRSDef: string;
+}
+
+/** Information returned from a request to interpret or complete a Geographic CRS
+ * @beta
+ */
+export interface GeographicCRSInterpretResponseProps {
+  status: number;
```

Comment:
Can this be more specific than `number`? What are valid values? How are they to be interpreted by caller?

Reply:
The current WKT parser only returns either ERROR (32768) or a csmap error (for which the list is both too exhaustive to provide any meaningful information). I can change the native so that either 0 (SUCCESS) or 32768 (ERROR) is returned and change the Typescript part. The WKT is in the process of being replaced and will return meaningful errors that I can add afterwards.

Reply:
Thanks. For now I'd just document that any status other than 0 indicates an error.

Reply:
Documented the request and response props.
I also noticed the geographicCRS prop could be undefined in the response so set it so.